radix/get_test.go

```diff
@@ -1,24 +1,33 @@
 package radix_test
 
 import (
-	"testing"
 	"github.com/SokolovVadim/Radix-Tree"
+	"testing"
+)
+
+const (
+	Length = 65536;
```

Comment:
why do you need to export the constants?

Reply:
fixed